handler.js

```diff
@@ -1,56 +1,56 @@
-let codeResponse = require("./utils/statusCode");
+const codeResponse = require("./utils/statusCode");
 
-module.exports.main = (event, context, callback) => {
-    let { transactions } = JSON.parse(event.body);
```

Comment:
Che, creo que esta variable no se está re-asignando, podrías cambiarla a una constante, lo cual ayuda a Node a optimizar el uso de su memoria.

Por ejemplo:
```javascript
const { transactions } = JSON.parse(event.body);
```

Acá podes leer un poco más: 
https://appdividend.com/2020/08/28/javascript-const-vs-let-the-complete-guide/#:~:text=JavaScript%20const%20vs%20let%20The%20%60const%60%20identifier%20is,console.log%20%28dana%29%20dana%20%3D%20%27Fox%20mulder%27%20console.log%20%28dana%29